Modules/Core/Common/include/itkIndexRange.h

```diff
@@ -459,7 +457,16 @@ using ImageRegionIndexRange = IndexRange<VDimension, false>;
 template <unsigned VDimension>
 using ZeroBasedIndexRange = IndexRange<VDimension, true>;
 
+
+#if !defined(ITK_LEGACY_REMOVE)
+namespace Experimental // For ITK 5.0 legacy support
```

Comment:
Some of these were introduced with 5.1, right? So the comment could be changed to indicate that.

Reply:
`ImageRegionRange` was introduced with ITK 5.1, indeed. Very well observed, @dzenanz The other ranges were already in ITK 5.0

My intention for now was to just remove all `#if !defined(ITK_LEGACY_REMOVE)` stuff that I originally added with this PR. Including the comments. OK?

Reply:
Sounds good. I was not sure whether you would keep `#ifdef`s.